webapp/src/main/webapp/src/assets/i18n/en.json

```diff
@@ -30,16 +30,19 @@
       }
     },
     "groups": {
-      "title": "Groups",
+      "title": "Student Group Administration",
       "select-school": "School",
       "select-school-year": "School Year",
       "select-subject": "Subject",
-      "no-results": "No groups found",
+      "no-results": "No groups found matching the given filter criteria.",
+      "no-schools": "Your user account does not have access to view the groups of any schools.",
```

Comment:
Looking for feedback on this message.  Technically it would be a bad setup for you to have a `GROUPS_READ` permission and have no schools scoped to that permission.